src/source/source_cache.js

```diff
@@ -704,14 +704,16 @@ class SourceCache extends Evented {
         for (let i = 0; i < ids.length; i++) {
             const tile = this._tiles[ids[i]];
             const tileID = tile.tileID;
+            const scale = Math.pow(2, this.transform.zoom - tile.tileID.overscaledZ);
+            const additionalRadius = tile.additionalRadius * EXTENT / tile.tileSize / scale;
```

Comment:
Can't getIds() return tiles that haven't loaded yet, in which case `tile.additionalRadius` will be undefined, and this expression will result in `NaN`?

Reply:
Yeah - I think it makes sense to initialize `Tile#additionalRadius` to `0` in the `Tile` constructor. (And if we weren't going to do that, we'd probably want to type it `?number` -- this is a common issue in our flow typing)